core/src/main/java/com/datastax/oss/driver/internal/core/adminrequest/AdminRequestHandler.java

```diff
@@ -148,7 +163,9 @@ public void onResponse(Frame responseFrame) {
       ByteBuffer pagingState = rows.getMetadata().pagingState;
       AdminRequestHandler nextHandler = (pagingState == null) ? null : this.copy(pagingState);
       setFinalResult(new AdminResult(rows, nextHandler, channel.protocolVersion()));
-    } else if (message instanceof Prepared) {
+    } else if (message instanceof Prepared
```

Comment:
I think we can broaden the scope of this check to `message instanceof Result` – `Rows` being just a special case of `Result` that was already handled in the previous `if` block.

Reply:
Good idea - taking into account the fact `Void` and `Prepared` is extending `Result`.
But the question is: do we want to treat `SchemaChange` and `SetKeyspace` as successes here as well? (both are extending `Result`). I think that we can, wdyt?

Reply:
The case has never presented itself until now (only `Rows` and `Prepared` were handled so far), so I assume that we can safely consider these as successes as well.

Reply:
ok, done